Source/Communication/LogWatcher.cs

```diff
@@ -31,7 +31,14 @@ public LogWatcher()
 #if DEBUG && DEBUG_TEST_LOG
 			_logPath = @"Test\FastBuildLog.log";
 #else
-			_logPath = Path.Combine(Path.GetTempPath(), LogRelativePath);
+			string path = Path.GetTempPath();
+			string fastbuildTempPath = System.Environment.GetEnvironmentVariable("FASTBUILD_TEMP_PATH");
+			if (fastbuildTempPath != null && System.IO.Directory.Exists(fastbuildTempPath))
```

Comment:
Prefer namespace importing over fully qualified type names
```suggestion
			if (fastbuildTempPath != null && Directory.Exists(fastbuildTempPath))
```